Doc improvements (#589)
pyansys/pyaedt: doc/source/API/Constants.rst

Constants
==========
This section lists constants that are commonly used in PyAEDT.

Example of constants usage:

.. code:: python

    from pyaedt import constants
    ipk = Icepak()
    # Use of AXIS Constant

>     cyl = ipk.modeler.primitives.create_cylinder(constants.AXIS.X, [0,0,0],10,3)
>     # Use of PLANE Constant
>     ipk.modeler.split(cyl, constants.PLANE.YZ, sides="Both")

Reviewer: ```suggestion
    cylinder = ipk.modeler.primitives.create_cylinder(constants.AXIS.X, [0,0,0],10,3)
    # Use of PLANE Constant
    ipk.modeler.split(cylinder, constants.PLANE.YZ, sides="Both")
```

==========

> This section lists all the useful constants used in pyaedt.

Reviewer 2: ```suggestion
This section lists commonly used constants in PyAEDT.
```
